libr/core/cmd_anal.c

```diff
@@ -9065,10 +9065,8 @@ static bool is_apple_target(RCore *core) {
 		return false;
 	}
 	RBinObject *bo = r_bin_cur_object (core->bin);
-	if (bo && bo->plugin && bo->plugin->name) {
-		return strstr (bo->plugin->name, "mach");
-	}
-	return false;
+	r_return_val_if_fail (bo && bo->plugin && bo->plugin->name, false);
```

Comment:
```suggestion
	r_return_val_if_fail (!bo || (bo->plugin && bo->plugin->name), false);
```

otherwise if `bo` is NULL it will create a warning, while it is a valid case.

Reply:
my bad